.gitignore

```diff
@@ -7,3 +7,4 @@ bin/
 .idea/
 out/
 ci/pipeline.yml
+.DS_Store
```

Comment:
Please move this to your global `.gitignore` list.

https://stackoverflow.com/questions/7335420/global-git-ignore